signer/src/storage/model.rs

```diff
@@ -310,9 +312,8 @@ pub struct WithdrawalRequest {
     pub sender_address: StacksPrincipal,
     /// The block height of the bitcoin blockchain when the stacks
     /// transaction that emitted this event was executed.
-    #[sqlx(try_from = "i64")]
```

Comment:
QQ: Couldn't we have implemented `TryFrom<i64>` for the newtypes and kept this, and skipped the custom impls in `sqlx.rs`?